src/test/java/com/epam/tc/hw2/ExerciseOneTest.java

```diff
@@ -0,0 +1,86 @@
+package com.epam.tc.hw2;
+
+import java.time.Duration;
+import java.util.List;
+import org.assertj.core.api.Assertions;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.annotations.Test;
+
+public class ExerciseOneTest extends AbstractTest {
+
+
+    private static String XPATH_ITEMS_IN_HEADER = "//ul[@class=\"uui-navigation nav navbar-nav m-l8\"]";
```

Comment:
This locator is not good. Because if one of these classes will be changed or change position in elements the locator won't work fine. So, it's unstable. Remove some extra classes from the locator